teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+class Teacher < Person
+  def initialize(name, especialization, age = 'Unknown', permission: true)
+    super(name, age, permission: permission)
+    @especialization = especialization
```

Comment:
- [ ] It would be better to  delete the `e` before specialization, please fix it  👍🏿

Reply:
Thanks for the advice, I will remove the `e`